handlers/gate/step.yml

```diff
@@ -0,0 +1,13 @@
+title: Gate
```

Comment:
I'm not sure if Gate is a well chosen name. It is very general. Based on the description and functionality, it could be  `Reject Files` or something along that line.